### ISSUE TITLE
vertical split of plot

### ISSUE THREAD

**Issue by xy12180** [sqlitebrowser/sqlitebrowser#2220]
Describe the new feature
--------------------------
In the current plot window, the columns and the plot are split horizontally. It would be better to have an option to split them vertically. Given tens of columns, this will save time to scroll down the screen to pick columns to plot.

Additional info
---------------
Please answer these questions before submitting your feature request.

#### Is your feature request related to an issue? Please include the issue number.
No.

#### Does this feature exist in another product or project? Please provide a link.
No.

#### Do you have a screenshot? Please add screenshots to help explain your idea.
columns
---------          ===>      columns   |    plot
   plot

**Comment by scottfurry** [comment]
I _think_ what you are describing is something akin to "table transforming" where the Y/X axis are swapped. Does this this sound right to you? I want to believe that's possible as a "transform" ability exists, IIRC and I'm not mixing libraries in my head.

**Comment by chrisjlocke** [comment]
I read it as more of a screen design thing so picking the columns was easier....

**Comment by xy12180** [comment]
Just a simple change, like what Chris said.

**Comment by mgrojo** [comment]
Do you mean, the table (Columns, X, Y1, Y2, Axys Type) on the left side and the Plot area on the right side? Having an option for that would make sense.

However, currently you can detach the Plot dock window and make it bigger. Maybe it already suits your needs.

**Comment by xy12180** [comment]
Yes, the table columns on the left side and plot on the right. Since the tables each often has hundreds of columns, even detaching the plot dock does not use the space efficiently.

For better screen space use, I think left/right split should be the default, instead of up/down.
